Agid r anaclerio

AnaclerioRiccardo opened the pull request:
Aggiunto nei sequence diagram SCIA001 e RI_InformationDebit le operazioni necessarie a gestire l'esito di una richiesta di conformazione.

AnaclerioRiccardo committed:
Update SCIA-001.mermaid

Aggiornato sequence diagram SCIA con esito conformazione

AnaclerioRiccardo committed:
Update SCIA-001.mermaid

Aggiornato sequence diagram SCIA con esito conformazione

AnaclerioRiccardo committed:
Update SCIA-001.mermaid

AnaclerioRiccardo committed:
Update SCIA-001.mermaid

AnaclerioRiccardo committed:
Update SCIA-001.mermaid

AnaclerioRiccardo committed:
Update SCIA-001.mermaid

AnaclerioRiccardo committed:
EsitoConformazione nel debito informativo

Modificato sequence diagram del RI_InformationDebits con l'inoltro dell'esito della conformazione a Registro Imprese

AnaclerioRiccardo committed:
EsitoConformazione nel debito informativo

Modificato sequence diagram del RI_InformationDebits con l'inoltro dell'esito della conformazione a Registro Imprese

AnaclerioRiccardo committed:
sequence diagram SCIA

Aggiunta notifica agli ET che è stata richiesta ad una conformazione

AnaclerioRiccardo committed:
sequence diagram SCIA

Aggiunta notifica agli ET che è stata richiesta ad una conformazione

AnaclerioRiccardo committed:
Update RI_InformationDebit-001.mermaid

AnaclerioRiccardo committed:
Update RI_InformationDebit-001.mermaid

AnaclerioRiccardo committed:
Aggiunto tempo di conformazione

Aggiunto tempo di conformazione

AnaclerioRiccardo committed:
Update SCIA-001.mermaid

AnaclerioRiccardo committed:
Update SCIA-001.mermaid

FCiminelliACN committed:
corretto la nota in merito alla gestione delle richieste di integrazione

FCiminelliACN committed:
corretto la nota in merito alla gestione delle richieste di integrazione

AnaclerioRiccardo committed:
Update RI_InformationDebit-001.mermaid

AnaclerioRiccardo committed:
Update RI_InformationDebit-001.mermaid

AnaclerioRiccardo committed:
Merge branch 'AGID-RAnaclerio' of https://github.com/AgID/specifiche-tecniche-DPR-160-2010 into AGID-RAnaclerio

AnaclerioRiccardo committed:
Merge branch 'AGID-RAnaclerio' of https://github.com/AgID/specifiche-tecniche-DPR-160-2010 into AGID-RAnaclerio

AnaclerioRiccardo committed:
Update SCIA-001.mermaid

aggiunta la risorsa alle notify di richiesta conformazione e di conformazione con esito negativo

AnaclerioRiccardo committed:
Update SCIA-001.mermaid

aggiunta la risorsa alle notify di richiesta conformazione e di conformazione con esito negativo

AnaclerioRiccardo committed:
Update SCIA-001.mermaid

AnaclerioRiccardo committed:
Update SCIA-001.mermaid

AnaclerioRiccardo committed:
Update SCIA-001.mermaid

Aggiunto riferimento all'ente terzo mittente nei messaggi di audit di conformazione

AnaclerioRiccardo committed:
Update SCIA-001.mermaid

Aggiunto riferimento all'ente terzo mittente nei messaggi di audit di conformazione

AnaclerioRiccardo committed:
Update times-schema.yaml

AnaclerioRiccardo committed:
Update times-schema.yaml

Rimosso temporaneamente tempo di conformazione, le OpenAPI saranno modificate con l'esito della conformazione una volta ricevuta l'approvazione delle modifiche apportate al BPMN e ai sequence diagram

AnaclerioRiccardo committed:
bpmn scia semplice

modificato il bpmn inserendo l'esito della conformazione

AnaclerioRiccardo committed:
Rinomina file

Rinominati i file ed eliminate cartelle superflue per ricondurre il nome dei file e la struttura a quanto presente nel branch main

FCiminelliACN committed:
Fix Issue 66 - timer conformazione ET e aggiornamento delle immagini e dei relativi openApi

FCiminelliACN committed:
Fix Issue 66 - aggiornamento stato InstanceDescriptor integrando terminazione della conformazione con esito positivo o negativo

FCiminelliACN committed:
Fix Issue 66 - aggiunta notifica al presentatore dopo lo scadere dei tempi e modificato oggetto e descrizione per il messaggio di conformazione

FCiminelliACN committed:
Fix Issue 66 - aggiornato workflow scia in comunica

FCiminelliACN committed:
Fix issue 66 - pausa e riavvio tempi procedimentali su richiesta di conformazione

FCiminelliACN committed:
Fix issue 66 - modifica evento audit finale

FCiminelliACN committed:
Fix issue 66 - modifica eventi audit verso il catalogo

FCiminelliACN committed:
Fix issue 130 - introduzione update_times con nota a seguito della ricezione di una richiesta di Conformazione/Integrazione o dell'istanza conformata/integrata

FCiminelliACN commented:
https://github.com/AgID/specifiche-tecniche-DPR-160-2010/issues/66
https://github.com/AgID/specifiche-tecniche-DPR-160-2010/issues/130

AnaclerioRiccardo committed:
Merge branch 'mantaince002' into AGID-RAnaclerio

AnaclerioRiccardo committed:
Merge branch 'AGID-RAnaclerio' of https://github.com/AgID/specifiche-tecniche-DPR-160-2010 into AGID-RAnaclerio

AnaclerioRiccardo committed:
Modello ER

Modificato il modello ER per spostare la relazione del regime amministrativo con il procedimento alla fattispecie

FCiminelliACN committed:
rimosso discriminator

AnaclerioRiccardo committed:
Ripristino modello ER

Ripristino del modello ER presente in approved01

FCiminelliACN committed:
Merge branch 'mantaince002' into AGID-RAnaclerio
